language-design.md

```diff
@@ -62,7 +175,50 @@ the commenters?
 
 **Response**
 
-
+Pavlus is arguing that programming languages should be designed to be easier
+for the average person to understand and learn. While we agree with this
+general argument of the article, the actual specifics Pavlus uses to bolster
+his claim are not convincing. The commenters point out that Pavlus is
+emphasizing the difficulty of learning a language for those new to the
+language, but they argue that a programming language should be designed more
+with its users in mind, in that ease of use should be the concern, not ease
+of learning.
+
+We mostly agree with the commenters. The language Pavlus seems to be after is a
+language for teaching programming, not one for actually programming. The reason
+that his "hieroglyphic" programming languages stick around is that they are
+useful. The commenters are right that there is a trade-off between a language
+being readable by the general public and how efficient and powerful a language
+is. We resent Pavlus for implying that such a trade-off need not be there, and
+that contributors to programming language design don't care enough to make their
+languages usable. Does he think that nobody has thought of this before?
```

Comment:
he might be right at some point down the road, maybe we won't have to trade off readability for efficiency and power?

we still do in 2015 though :(